### PR TITLE
Bug/homepage not displaying cookie banner

### PR DESCRIPTION
**Pull request by liamjlewis**
### What
Trello card https://trello.com/c/1NbkONTs/961-homepage-is-not-displaying-the-cookie-banner

The cookies message has previously been removed from the homepage with a view to using dp-renderer, however the homepage has not yet been updated to use dp-renderer rather than dp-frontend-renderer. After some discussion a temporary solution was decided on of reinstating the old cookie. An HTML tag of "data-nosnippet" has been added to stop google from using the cookie message in the google search preview.

### How to review

- Check the code reverted to doesn't accidentally include anything unnecessary. 
- Check the cookie banner is loading locally.

### Who can review

Anyone.

### CODE REVIEW

File: assets/locales/active.cy.toml [hunk 1]
```diff
@@ -640,4 +783,4 @@ one = "Did you know?"
 
 [ProduceStatisticsRelatingTo]
 description = "also produce statistics relating to"
-one = "also produce statistics relating to" 
+one = "also produce statistics relating to"
```
Comment (NoelleDL) [inline assets/locales/active.cy.toml:786]: New line needed

File: assets/templates/main.tmpl [hunk 2]
```diff
@@ -39,6 +39,10 @@
   <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MBCBVQS"
                     height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
   <!-- End Google Tag Manager (noscript) -->
+    {{ if (ne .FeatureFlags.HideCookieBanner true)}}
+      {{ template "partials/banners/cookies" . }}
```
Comment (NoelleDL) [inline assets/templates/main.tmpl:43]: Believe this should be {{ if not .FeatureFlags.... }}

File: assets/templates/partials/banners/cookies.tmpl [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                        <p class="cookies-banner__confirmation-message">
+                            {{ localise "CookiesBannerSuccess" .Language 1 | safeHTML }}
+                            <button type="button" class="cookies-banner__button--hide js-hide-cookies-banner">{{ localise "CookiesBannerHide" .Language 1 }}</button>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </form>
+</section>
```
Comment (NoelleDL) [inline assets/templates/partials/banners/cookies.tmpl:34]: New line needed

File: assets/templates/main.tmpl (hunk 2 above)
Comment (liamjlewis) [reply assets/templates/main.tmpl:43]: I'm still getting familiar with Go, should the syntax be:
`{{ if not .FeatureFlags.HideCookieBanner true}}`

Comment (liamjlewis) [reply assets/templates/main.tmpl:43]: I'm still getting familiar with Go, should the syntax be:
`{{ if not .FeatureFlags.HideCookieBanner true}}`

Comment (liamjlewis) [reply assets/templates/main.tmpl:43]: or this:
`{{ if ne FeatureFlags.HideCookieBanner true }}`

**Pull request closed without merging**